src/universal/modules/meeting/helpers/handleRedirects.js

```diff
@@ -86,6 +87,9 @@ export default function handleRedirects(team, localPhase, localPhaseItem, oldTea
       return false;
     }
   }
+  if (team.facilitatorPhase === SUMMARY) {
```

Comment:
Currently, this won't ever be reached because `endMeeting` will reset the facilitatorPhase to `lobby`.